scripts/generateAssetData/generateAssetData.ts

```diff
@@ -28,6 +30,8 @@ import { overrideAssets } from './overrides'
 import * as polygon from './polygon'
 import { filterOutBlacklistedAssets } from './utils'
 
+const clearAssetsCodemodCommand = 'yarn run codemod:clear-assets-migration'
```

Comment:
i would get rid of pify and the manual exec and just chain the yarn commands in package.json

Reply:
🧠

Reply:
ac6ee8eb9f4c4d2341b4e42cd802ec65ac507f8a